modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ocp_{context}"]
+= Manually configuring {ocp} registry
+
+You can integrate {product-title} with {ocp} built-in container image registry.
+
+.Prerequisites
+* You need a *Username* and a *Password* for authentication with the {ocp} registry.
```

Comment:
I don't see why these aren't styled as general text--username and password. I think the context is "have them ready" rather than "enter them in to the UI fields" that comes later. If this project has a distinct style, then ignore.